plugins/LadspaEffect/caps/basics.h

```diff
@@ -76,7 +79,7 @@
 
 #define MIN_GAIN .000001 /* -120 dB */
 
-/* smallest non-denormal 32 bit IEEE float is 1.18�10-38 */
+/* smallest non-denormal 32 bit IEEE float is 1.18×10-38 */
```

Comment:
Yeah, I believe your PC is converting the UTF8 to Latin-1.

If you look at this chart (bottom line), you can see why this occurs:
https://www.cogsci.ed.ac.uk/~richard/utf-8.cgi?input=%C3%97&mode=char

With that understood, lets use the lower-case `x` instead to guard this for future contributors.  I think Tim will see that in the diff and immediately understand.  I do not think he'll make the same assumption with the asterisk, since it's not immediately obvious.